integration/helpers/pepr.ts

```diff
@@ -0,0 +1,50 @@
+import { resolve } from "node:path";
+import { cp, readFile, writeFile } from "node:fs/promises";
+import { basename } from "node:path";
+import { Spec, Cmd, Result } from "./cmd";
+import { clone } from "ramda";
+
+const HERE = __dirname;
+
+export async function projectRoot(): Promise<string> {
+  const cmd = new Cmd({ cmd: `npm root`, cwd: HERE });
+  const res = await cmd.run();
+  const npmroot = res.stdout.join("").trim();
+  return resolve(npmroot, "..");
+}
+
+export async function tgzifyModule(modulePath: string): Promise<void> {
+  const packagePath = `${modulePath}/package.json`;
+  const packageJson = JSON.parse(await readFile(packagePath, { encoding: "utf8" }));
+
+  const root = await projectRoot();
+  packageJson.dependencies.pepr = `file://${root}/pepr-0.0.0-development.tgz`;
+  await writeFile(packagePath, JSON.stringify(packageJson, null, 2));
+}
+
+export async function copyModule(src: string, dst: string): Promise<void> {
```

Comment:
`copyModule()` is only called once by `pepr.test.ts` and not used elsewhere. Do we really need it?

Reply:
Nope -- you're right.  This one was an oversight on my part when I did the conversion from serialized-to-parallelizable test layout I just fogot to remove it.

Consider it gone.